test/test_product_config.py

```diff
@@ -528,14 +528,21 @@ def test_from_config_narrowband_vlbi(
         assert acv.window_function == defaults.WINDOW_FUNCTION_VLBI
         assert acv.pfb_taps == 1
 
+    @pytest.mark.parametrize("extra", [0.0, 1e6])
     def test_vlbi_bad_pass_bandwidth(
         self,
         narrowband_vlbi_config: Dict[str, Any],
         src_streams: List[DigBasebandVoltageStreamBase],
+        extra: float,
     ) -> None:
         bandwidth = src_streams[0].adc_sample_rate / 2 / DECIMATION
-        narrowband_vlbi_config["narrowband"]["vlbi"]["pass_bandwidth"] = bandwidth
-        with pytest.raises(ValueError):
+        pass_bandwidth = bandwidth + extra
+        narrowband_vlbi_config["narrowband"]["vlbi"]["pass_bandwidth"] = pass_bandwidth
+        with pytest.raises(
+            ValueError,
+            match=rf"pass_bandwidth \({pass_bandwidth}\) must be strictly less than "
+            rf"bandwidth \({bandwidth}\)",
```

Comment:
Interesting you use `rf` but also include the backslashes for the normal brackets/braces.

I see Line 1161 (`test_bad_continuum_factor`) also uses `rf"..."` but doesn't include the backslashes for its normal braces. Either approach works? One more sound than the other?

Reply:
The difference is line 1161 wrapped the string in `re.escape` to escape the parentheses, rather than escaping them with backslashes. I like that, so I've made the same change here.

The r prefix means that backslashes are regex escapes instead of Python string escapes. With the backslashes gone it's not needed, so I've removed it.

Reply:
My apologies, you're quite right Re: `re.escape`. Still, appreciate the update/conformity.